src/main/java/com/prgrms/board/error/ValidErrorResponse.java

```diff
@@ -0,0 +1,11 @@
+package com.prgrms.board.error;
+
+import lombok.Builder;
+
+public record ValidErrorResponse(String field, String message) {
```

Comment:
```suggestion
public record InvalidErrorResponse(String field, String message) {
```
validityerror, validationerror 등등 다양한게 될 수 있지만 validError는 어감이 조금 이상한거 같아요 🤔

Reply:
InvalidErrorResponse 로 반영하였습니다!